### ISSUE TITLE
Empty versions

### ISSUE THREAD

**Issue by avoidik** [cronn-de/jira-sync#2]
Good day.
Very good project, thank you for your effort.
Found issue when trying to get this project started, with empty `versions` objects a target server rejects synchronizations with error
```
Bad Request: {"errorMessages":[],"errors":{"versions":"Field 'versions' cannot be set. It is not on the appropriate screen, or unknown."}}
```
```
"versions": [],
```

Use case
New unassigned items like stories in a backlog (Scrum project)

**Comment by avoidik** [comment]
Set missing fields in Issues > Screens in Jira for you project

**Comment by ldziedziul** [comment]
Hi @avoidik, I think it has been fixed along with #4

**Comment by avoidik** [comment]
In my case it was mismatch between [Issues Screens](https://confluence.atlassian.com/adminjiraserver071/project-screens-schemes-and-fields-802592517.html), but anyway I'm glad to hear that such edge case has been fixed.
